src/main/kotlin/mate/academy/Main.kt

```diff
@@ -1,7 +1,14 @@
 package mate.academy
 
 fun greetUsers() {
-    // implement function
+    while(true) {
+        println("Please enter a name (or press Enter to quit): ")
+        val name = readLine()
+        if (name?.length == 0) {
```

Comment:
Could be simplified with Kotlin's functionality.
Please find more info here
https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.collections/is-null-or-empty.html
https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.collections/if-empty.html